pkg/daemon/daemon.go

```diff
@@ -246,7 +248,9 @@ func (d *daemon) AddPeriodicUpdate() {
 					d.guidPodNetworkMap[allocatedGUID] = podNetworkID
 				}
 
-				if err = utils.SetPodNetworkGUID(network, allocatedGUID); err != nil {
+				// TODO(adrianc): should we just set GUID both in CNI ARGS and runtime config ?
```

Comment:
why it should be both?

Reply:
i was thinking at the time of a cluster with different versions of ib-sriov cni where some might support infinibandGUID runtime config and some might not, however we should not hide configuration issues here so ill remove this TODO.